bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/BigtableDataGrpcClient.java

```diff
@@ -77,7 +74,8 @@
       new Predicate<MutateRowRequest>() {
         @Override
         public boolean apply(MutateRowRequest mutateRowRequest) {
-          return allCellsHaveTimestamps(mutateRowRequest.getMutationsList());
+          return mutateRowRequest != null
```

Comment:
nit: I guess I would prefer if the tests didn't use null. Seems wrong to shortcircuit the code for the tests. Up to you.